tests/node_service/index.mjs

```diff
@@ -0,0 +1,68 @@
+import { randomUUID } from 'crypto'
```

Comment:
Is this used?

Reply:
Oh it's not anymore 🤔 I did have it previously and decided it was pointless. I'll remove this